scipy/spatial/transform/_rotation.pyx

```diff
@@ -2381,16 +2385,100 @@ cdef class Rotation:
+
         if not all(isinstance(x, Rotation) for x in rotations):
             raise TypeError("input must contain Rotation objects only")
 
         quats = np.concatenate([np.atleast_2d(x.as_quat()) for x in rotations])
         return cls(quats, normalize=False)
 
+    def split(self, indices_or_sections):
+        """Split a `Rotation` object into a sequence of `Rotation` objects.
+
+        This matches the syntax of
+        [numpy.split](https://numpy.org/doc/stable/reference/generated/numpy.split.html),
+        and is an inverse of `concatenate`. To split out all rotations from
+        a `Rotation` object `R`, use `indices_or_sections` equal to the number
+        of rotations it contains: `R.split(len(R))`.
+
+        Parameters
+        ----------
+        indices_or_sections : int or 1-D array
+            If `indices_or_sections` is an integer, ``N``, the array will be
+            divided into ``N`` equal length rotations. If such a split is not
+            possible, an error is raised. The identity ``N == 1`` will return a
+            copy of the original object.
+
+            If `indices_or_sections` is a 1-D array of sorted integers, the
+            entries indicate where the rotation is split. For example,
+            ``[2, 4]`` would result in
+
+            - R[:2]
+            - R[2:4]
+            - R[4:]
+
+            If an index exceeds the length of `R`, it is ignored.
+
+        Returns
+        -------
+        rotations : `Rotation` or list of `Rotation` objects
+            The split rotations. If the input contains a single rotation or
+            ``N == 1``, a single `Rotation` object is returned. Otherwise, a
+            list of `Rotation` objects is returned.
+
+        Notes
+        -----
+        .. versionadded:: 1.15.0
+
+        Examples
+        --------
+        The most common use is a split into individual rotations.
+
+        >>> from scipy.spatial.transform import Rotation as R
+        >>> r0 = R.from_rotvec([[0, 0, 0], [0, 0, 1], [0, 0, 2], [0, 0, 3]])
+        >>> rs = r0.split(len(r0))
+        >>> print(*[r.as_rotvec() for r in rs])
+        array([0., 0., 0.])
+        array([0., 0., 1.])
+        array([0., 0., 2.])
+        array([0., 0., 3.])
+
+        Here is a more complex split at the 1st and 2nd indices.
+
+        >>> rs = r0.split([1, 2])
+        >>> print(*[r.as_rotvec() for r in rs])
+        array([0., 0., 0.])
+        array([0., 0., 1.])
+        array([[0., 0., 2.], [0., 0., 3.]])
+
+        Concatenation of a split rotation recovers the original object.
+
+        >>> print(R.concatenate(rs).as_rotvec())
+        array([[0., 0., 0.],
+               [0., 0., 1.],
+               [0., 0., 2.],
+               [0., 0., 3.]])
+        """
+        Rs = [self.__class__(np.squeeze(qs), normalize=False)
+                for qs in np.split(self.as_quat(), indices_or_sections)]
+        if len(Rs) == 1:
+            return Rs[0]
```

Comment:
I think there is a debatable API choice in returning a Rotation object directly (instead of a list with that object) if there is a only a single Rotation in the split list. This happens if splitting a single rotation, or splitting with the entire list of rotations in a single group (essentially an identity operation). Rotation.concatenate has been updated to allow taking in a single Rotation object as well, to match behavior. Thoughts?